packages/server/tests/localAcceptance.env

```diff
@@ -3,6 +3,8 @@ OPERATOR_ID_MAIN=0.0.2
 OPERATOR_KEY_MAIN=302e020100300506032b65700422042091132178e72057a1d7528025956fe39b0b847f200ab59b2fdd367017f3087137
 CHAIN_ID=0x12a
 MIRROR_NODE_URL_WEB3=http://127.0.0.1:8545
+REDIS_ENABLED=true
```

Comment:
does this needs redis service to be running on `hedera-local-node` to work properly?

Reply:
No, only requirement is REDIS_URL to be pointed to correct address. But for acceptance test we will use hedera-local-node redis container.